win32port/win32helpers/gettimeofday.c

```diff
@@ -1,5 +1,5 @@
 #include <time.h>
-#include <windows.h> //I've omitted context line

```

Comment:
Since you change this. When building normal windows build, first the `timezone` struct is defined in the header, and then it's defined in here again. which breaks. This needs to be fixed so it works for both MINGW and normal